tcms/testruns/static/testruns/js/get.js

```diff
@@ -4,6 +4,15 @@ $(document).ready(() => {
 
     const testRunId = $('#test_run_pk').data('pk')
 
+    $('#status_button').on('switchChange.bootstrapSwitch', (_event, state) => {
+        if (state) {
+            jsonRPC('TestRun.update', [testRunId, { 'stop_date': null }], () => { })
+        } else {
+            const now = new Date()
+            jsonRPC('TestRun.update', [testRunId, { 'stop_date': `${now.getUTCFullYear()}-${now.getUTCMonth()}-${now.getUTCDay()} [${now.getUTCHours()}:${now.getUTCMinutes()}:${now.getUTCSeconds()}]}` }], () => { })
```

Comment:
Isn't `.toISOString()` easier ?

From the rest of the discussion it sounds like this should be a helper function which returns the current timestamp, appropriately formatted and adjusted for TZ offsset if necessary.

Reply:
`toISOString()` returns `"2020-04-13T14:45:21.125Z"` which is not compatible with the format we support on the BE

Reply:
Replace "T" with " " and cut the last 5 chars and it is

Reply:
done. do you want me to extract it into a method in `utils.js`?

Reply:
actually, this is wrong. the way it is now, it is not in UTC, but in the client time-zone. we need to use the `toUTC...` methods to achieve UTC conversion

Reply:
Correct, 1st need to convert the value to UTC and then format it. Let's do this next b/c there will be a few more items to it.